src/cloudant/error.py

```diff
@@ -16,6 +16,16 @@
 Module that contains common exception classes for the Cloudant Python client
```

Comment:
You should probably change your copyright notice to `Copyright (c) 2015 IBM. All rights reserved.` for this module.